docs/pages/admin-guides/management/operations/breakglass-access.mdx

```diff
@@ -0,0 +1,197 @@
+We will use this mechanism to restrict cert-based SSH access only to the `breakglass` user to prevent other Teleport users from being able to generate client certificates and gain access to the server outside of Teleport.
+
+1. Ensure the following line is added and/or uncommented in your `/etc/ssh/sshd_config` file:
+
+```
+AuthorizedPrincipalsFile /etc/ssh/authorized_principals/%u
+```
+
+2. Create an authorized_principals file for the `breakglass` user 
+
+```bash
+echo "breakglass" | sudo tee /etc/ssh/authorized_principals/breakglass
+```
+
+3. Restart the sshd service:
+
+```bash
+sudo systemctl restart sshd
+```
+
+## Step 4/6. Create `breakglass` Role and User in Teleport for impersonation
+
+1. Create `breakglass` Teleport role:
+
+Define the role in a file named breakglass-role.yaml
+
+```yaml
+kind: role
+metadata:
+  name: breakglass
+spec:
+  allow:
+    logins:
+    - breakglass
+    node_labels:
+      '*': '*'
+  deny: {}
+  options:
+    cert_format: standard
+    forward_agent: false
+    max_session_ttl: 24h0m0s
+    port_forwarding: true
+    ssh_file_copy: true
+version: v7
+```
+
+Create the role by applying with:
+
+```bash
+tctl create -f breakglass-role.yaml
+```
+
+2. Create the `breakglass` Teleport user:
+
+```bash
+tctl users add --roles=breakglass breakglass
+```
+
+We don't need to worry about setting a password for this user as it'll only ever be used in the context of OpenSSH break glass authentication via impersonation.
+
+3. Grant `impersonate` permissions for `breakglass` User and Role:
+
+To avoid having to directly log into Teleport as `breakglass`, grant `impersonation` permissions to an admin or automation user by adding the following to the impersonating user's matching role:
+
+```yaml
+impersonate:
+    roles:
+    - breakglass
+    users:
+    - breakglass
+```
+
+## Step 5/6. Generate a `breakglass` ssh key and cert
+
+Run the following after logging into Teleport via `tsh login` as the impersonating user:
+
+```bash
+tctl auth sign --ttl 24h --user=breakglass --out=/safe/path/breakglass --format openssh
+```
+
+This command will create an ssh private key and a Teleport CA-signed certificate to be used by your OpenSSH client when authenticating with the OpenSSH server.
+
+Store the private key securely in a vault or other secure location, and only access if needed to implement break glass procedures. 
+
+<Admonition type="tip">
+If you've followed this guide, the resulting `breakglass` Teleport CA-signed cert will be valid only for 24 hours. To ensure that you have a valid and up-to-date certificate in case of an outage, regularly rotate both the key and certificate before they expire. Use [Machine ID](../../../../pages/enroll-resources/machine-id/machine-id.mdx) or automate renewal with a cron job or systemd timer.
```

Comment:
This is basically the most important part of the guide and yet it's left entirely to the user to figure out - machine ID is likely the correct solution here, but it won't need impersonation for a "service" account such as the `breakglass` user mentioned above. Securing such a bot would also become _really_ critical.